src/net/java/sip/communicator/service/protocol/StunServerDescriptor.java

```diff
@@ -248,8 +250,16 @@ public void storeDescriptor(Map<String, String> props, String namePrefix)
             props.put(namePrefix + STUN_PORT, Integer.toString( getPort() ));
 
         if (getUsername() != null && getUsername().length > 0)
-            props.put(namePrefix + STUN_USERNAME,
-                      StringUtils.getUTF8String(getUsername()));
+            try
+            {
+                props.put(namePrefix + STUN_USERNAME,
+                          StringUtils.toString(getUsername(), StandardCharsets.UTF_8.name()));
```

Comment:
use new String(..., StandardCharsets.UTF_8). Or if this throws, use toEncodedString